ddmrp_packaging_product_replace/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,7 @@
+This is a glue module between `ddmrp_packaging` and `ddmrp_product_replace`.
+It disable the copy of the packaging information when using the product
+replacement wizard.
+Without it the new stock buffer created have the packaging of the old product
+set on them and the constraint `_check_product_packaging` raises an error.
+
+Strangely, this is not happening with Odoo 13 ?
```

Comment:
Could it be related to the workaround applied by this v13 commit: https://github.com/OCA/ddmrp/commit/5c0acb3c
that is not required anymore or that requires change to work properly?

Reply:
That change has also been merged in 13.0 https://github.com/OCA/ddmrp/blob/e4a60f64c89304afac2353accc57af7d7101ac4f/ddmrp_packaging/models/stock_buffer.py#L18